torch/testing/_internal/common_modules.py

```diff
@@ -4273,20 +4288,28 @@ def module_error_inputs_torch_nn_Pad3d(module_info, device, dtype, requires_grad
                ),
     ModuleInfo(torch.nn.Tanh,
                module_inputs_func=module_inputs_torch_nn_Tanh,
-               skips=(
+               skips=None if _macos15_or_newer else (
                    # Fails on backward check on MPS
                    # See https://github.com/pytorch/pytorch/issues/107214
                    DecorateInfo(
-                       skipMPSVersionIfLessThan(15, 0)
+                       unittest.expectedFailure,
+                       'TestModule',
+                       'test_memory_format',
+                       active_if=operator.itemgetter('training'),
+                       device_type='mps',
                    ),)
                ),
     ModuleInfo(torch.nn.Tanhshrink,
                module_inputs_func=module_inputs_torch_nn_Tanhshrink,
-               skips=(
+               skips=None if _macos15_or_newer else (
```

Comment:
Do we also want the skips here to be None for non macos devices? like 
```suggestion
               skips=None if _macos15_or_newer or not is_macos else (
```
where is_macos is defined as being on macos at the time?

Reply:
It's a bit confusing, but it works as expected as skip is only for `device_type='mps'`

Reply:
Yea I saw that, I was more thinking about the unnecessary overhead